client_connection/manager.go

```diff
@@ -2,7 +2,7 @@ package client_connection
 
 import (
 	"errors"
-	"github.com/mysterium/node/service_discovery/dto"
+	id "github.com/mysterium/node/identity"
```

Comment:
confusing naming. Identity from "id" package :/ why we need alias here? (and in other imports)

Reply:
I just wanted to shorten it, plus there are quite a few variables called "identity" so it kind of collides with the package name. Didn't want to refactor too much.

Reply:
I did actually use `id` for variable names to avoid this clash in my code xD

Reply:
:D so should I change it? Already got a commit for it.

Reply:
Yeah, looks strange for me also

Reply:
Cool, this will be pushed in a bit.